src/Tools/ExternalAccess/Xaml/External/IXamlRequestHandler.cs

```diff
@@ -2,11 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.CommonLanguageServerProtocol.Framework;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.Xaml;
 
 internal interface IXamlRequestHandler<TRequest, TResponse>
-    : IRequestHandler<TRequest, TResponse, XamlRequestContext>
```

Comment:
This was incorrectly exposing the clasp type directly to XAML.  making it internal broke their handlers.  However XAML already wrapped this handler into a Roslyn typed handler (XamlRequestHandlerBase), so all we need to do here is copy the old signature directly here and remove the inheritance.